app/views/teaching_events/show/_how-to-attend.html.erb

```diff
@@ -11,13 +11,16 @@
 
     <% if can_sign_up_online?(@event) %>
       <p>
-        You must register for a place to attend this event.
-
+        Register for this event and we can offer further tailored support and advice on your journey into teaching.
```

Comment:
I'm wondering if we might want to be more specific here - to me, this kinda sounds like if you sign up for an event, you're essentially joining our mailing list?

Instead, for in-person events, could we say something like 'Register for this event and you'll receive a QR code to use for a quicker check-in process when you arrive, as well as reminder emails about events near you' or something along those lines?

(Sorry to complicate things!!)

Reply:
Thanks, I see your point and I did consider this but I don't think it's currently structured to show different content for in-person/online events. Is that possible @plisovin?